PHP/CodingConventions.md

```diff
@@ -0,0 +1,77 @@
+# Wikia PHP Coding Conventions
+
+This file documents Wikia's PHP coding conventions. To propose changes to this document, please open a pull request
+with your changes and tag [@engineers](https://github.com/orgs/Wikia/teams/engineers) for review.
+
+
+## TOC
+
+* [Base Rules](#base-rules)
+* [Additional Rules](#additional-rules)
+  * [Function Length](#function-length)
+  * [Type-Checking and Assertions](#type-checking-and-assertions)
+* [Tools](#tools)
+  * [MediaWiki PHP Style Helper](#mediawiki-php-style-helper)
+
+## Base Rules
+
+Wikia will continue to follow the MediaWiki style guidelines for PHP writing code. These guidelines for PHP can be
+found in the [MediaWiki Coding Conventions](http://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP) manual.
+
+## Additional Rules
+
+### Function Length
+
+Functions should have a limited length. This limit is loosely defined as one “page”, where a page is the number
```

Comment:
I know this is just a copy/paste but I'd like to see this rule tightened down to something more along the lines of "Functions should do only one thing. If a single function does more than one thing it should be broken up into smaller functions which the original function can call."